{
  "_annotation": {
    "cve_id": "CVE-2024-6923",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-6923.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "B077F990-E5D3-548E-B472-E262426A9F27",
                "versionEndIncluding": "3.13.0rc2",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}